cache/cache.go

```diff
@@ -109,9 +109,12 @@ func (r *RedisClient) SetJSON(key string, value interface{}, expiration time.Dur
 // If the client was set up with a prefix it will be added in front of the key.
 func (r *RedisClient) GetJSON(key string, result interface{}) error {
 	resultStr, err := r.Get(key)
-	if err != nil {
+	if err == redis.Nil {
+		return ErrNotFound
+	} else if err != nil {
```

Comment:
From what I can see this `else if err != nil` part is missing in functions like `GetBool`. Could you also fix that in this MR?